people.json

```diff
@@ -5774,22 +5774,24 @@
-      "company": "Logz.io",
+      "bio": "<p>Horovits lives at the intersection of technology, product and innovation. With over 20 years in the hi-tech industry as a software developer, a solutions architect and a product manager, he brings a wealth of knowledge in cloud and cloud native, DevOps practices, open source and more.</p><p>Horovits is a public speaker and thought leader in the fields of DevOps, open source and specifically observability, having been <a href=\"https://www.youtube.com/@horovits\">on stage</a> at KubeCon, Open Source Summit, FOSDEM and more. He also runs the popular <a href=\"https://podcasters.spotify.com/pod/show/openobservability\">OpenObservability Talks podcast</a>, and you can also follow his <a href=\"https://horovits.medium.com/\">blog</a> and articles.</p><p>Horovits has been a leader in the <a href=\"https://www.cncf.io/blog/2023/02/24/celebrating-israels-first-kubernetes-community-days-a-look-at-the-history-of-our-cncf-community/\">Israeli community</a>, an organizer of the CNCF chapter in Tel Aviv, and an organizer of the Israeli editions of Kubernetes Community Days (KCD), KubeDay and DevOpsDays.</p><p><ul><li>Podcast: <a href=\"https://podcasters.spotify.com/pod/show/openobservability\">podcasters.spotify.com/pod/show/openobservability</a></li><li>Blog: <a href=\"http:\\horovits.medium.com/\">medium.com/@horovits</a></li><li>YouTube: <a href=\"https://www.youtube.com/@horovits\">youtube.com/@horovits</a></li><li>Sessionize: <a href=\"https://sessionize.com/horovits\">sessionize.com/horovits</a></li></ul></p><p>Why I am excited to be an ambassador: I've been advocating for open source and for the cloud native stack for years, online, on stage and with local communities, and I'm looking forward to expanding this activity as a CNCF Ambassador.</p>",
+      "company": "",
       "pronouns": "He/Him",
       "location": "Tel Aviv, Israel",
       "linkedin": "https://www.linkedin.com/in/horovits/",
       "twitter": "https://www.twitter.com/horovits",
       "github": "https://logz.io/author/dotan-horovits/",
```

Comment:
This should be the link to your GitHub user

Reply:
This line is the company field, which I changed to an empty string, as I no longer work there. I saw that's the convention used on this file for that. 
Other than that, I only updated mastodon and wechat fields, rest remains as before.

Reply:
Right, I know you didn't update it recently, however, I just noticed that this line is not pointing to a GH user:
`"github": "https://logz.io/author/dotan-horovits/",`

It looks a bit odd on your profile since we use the GH icon:

<img width="300" src="https://github.com/user-attachments/assets/a6493504-3876-458c-a65f-2ff0f5388401">

Just thought you might want to fix it before merging this PR... but if not, no worries.

Reply:
good catch. just submitted a fix, and reorganized the social links so social media appear one after the other